packages/lit-dev-content/site/docs/ssr/lifecycle.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Lit SSR lifecycle
+eleventyNavigation:
+  key: Lifecycle
+  parent: Server rendering
+  order: 5
+---
+
+{% labs-disclaimer %}
+
+The tables below lists the standard custom element and Lit element lifecycle methods and whether they are called during SSR and during hydration.
+
+Be mindful that methods called on the server should not contain references to browser/DOM APIs that have not been shimmed. Methods that are not called server-side may contain those references without throwing.
+
+## LitElement
+| Method | Called on server | Notes |
+|-|-|-|
+| `constructor()` | ✅ | |
```

Comment:
Out of scope for this, but can you add a TODO here and in `dom-emulation.md` to migrate to the existing icons since they've been tested to work on dark backgrounds?

- check
  - https://github.com/lit/lit.dev/blob/main/packages/lit-dev-content/src/icons/green-check-icon.ts
- x
  - https://github.com/lit/lit.dev/blob/main/packages/lit-dev-content/src/icons/red-x-icon.ts
- warn
  - https://github.com/lit/lit.dev/blob/main/packages/lit-dev-content/src/icons/yellow-bang-icon.ts

Out of scope because we gotta figure out an image strategy here to not duplicate SVGs and reduce server requests. Maybe `components/lazy-svg` might be the solution here